models/comment.go

```diff
@@ -36,6 +36,15 @@ const (
 	COMMENT_TYPE_COMMENT_REF
 	// Reference from a pull request
 	COMMENT_TYPE_PULL_REF
+
+	// Labels
+	COMMENT_TYPE_LABEL
+	// Milestones
+	COMMENT_TYPE_MILESTONE
+	// Title
+	COMMENT_TYPE_TITLE
+	// Assignee
+	COMMENT_TYPE_ASSIGNEE
```

Comment:
Add `_CHANGE` suffix to these four new types is more accurate.

Reply:
Done.